assets/filtering.js

```diff
@@ -0,0 +1,20 @@
+export function makeGroupNamesFilters(groupNames) {
```

Comment:
Helper functions in vue apps go into a subfolder, like `assets/vue/helpers/filtering.js`.